content/en/docs/tutorials/_index.md

```diff
@@ -26,8 +23,8 @@ get to a final goal. Below is the list of tutorials that you might find helpful:
   certificate.
 
 
-User How-To Blog Posts and Examples:
+### External Tutorials
```

Comment:
Random thought: I wonder if it'd make sense to combine this section with the one below `Once an [`Issuer`](../configuration/) has been configured, you're ready to issue your first certificate!` since they both essentially contain various external tutorials

Reply:
I think that's a great suggestion, yeah; I think I'd favour delaying making that change though, since we're intending to work on the tutorials section soon as we've mentioned in standups + meetings. I think we can sort that out when we write the tutorials.